package.json

```diff
@@ -17,7 +17,13 @@
     "type": "git",
     "url": "git://github.com/Teamwork/node-afk.git"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@paulcbetts/system-idle-time": "^1.0.4"
+  },
   "devDependencies": {},
-  "main": "index.js"
+  "main": "index.js",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "tag": "beta"
```

Comment:
Because we don't wanna release a new version that people will update to until we're sure it's gonna work.